src/IndexingWrapper.cpp

```diff
@@ -242,40 +242,32 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
 	    std::string val;
 
 	    if (list.containsElementNamed("slot")) {
-		try {
 		    slot = Rcpp::as<int>(list["slot"]);
```

Comment:
What happens if someone puts something in `slot` that isn't an `int`? Having the `try…catch` before was presumably needed to deal with that case, so I'm not sure this is an improvement (and similarly throughout this commit).

The commit message is poor as well. Is it improving readability? It seems to be removing functionality. If it's genuinely unneeded functionality, then explain why in the commit message so others reading through can see what's going on. (Or you, at some point in the future, trying to remember why you made this change!)